include/svnlook.php

```diff
@@ -797,18 +797,26 @@ function applyGeshi($path, $filename, $language, $rev, $peg = '', $return = fals
 		}
 
 		$source = file_get_contents($filename);
+
 		if ($this->geshi === null) {
 			if (!defined('USE_AUTOLOADER')) {
 				require_once 'geshi.php';
 			}
 			$this->geshi = new GeSHi();
 		}
+
 		$this->geshi->set_source($source);
 		$this->geshi->set_language($language);
 		$this->geshi->set_header_type(GESHI_HEADER_NONE);
 		$this->geshi->set_overall_class('geshi');
 		$this->geshi->set_tab_width($this->repConfig->getExpandTabsBy());
 
+		if ($highlight == 'file') {
+			$this->geshi->enable_line_numbers(GESHI_FANCY_LINE_NUMBERS,1);
```

Comment:
The `1` doesn't add any benefit. It will just make every line bold.

Reply:
By default fancy line numbers are normal line numbers. Adding 1 as an argument makes them "fancy" or "bold". Initially i had not passed any arguments as i was hoping maybe in future we can somehow pass the line number in the url here. So only those line numbers would be marked as bold. Link in case i pass `#line-135` then only line 135 will be marked rest will be normal.

Reply:
> By default fancy line numbers are normal line numbers. Adding 1 as an argument makes them "fancy" or "bold". Initially i had not passed any arguments as i was hoping maybe in future we can somehow pass the line number in the url here. So only those line numbers would be marked as bold. Link in case i pass `#line-135` then only line 135 will be marked rest will be normal.

This is not what I see. The number means in what step numbers will be bold. I don't see a reavery to make every number bold.

Reply:
ahh okay. Anyways That is a topic for another PR. Completely dropped the 2nd argument.